### DISCUSSION TITLE
在RHEL8 安装 [1Panel Log]: docker 失败

### DISCUSSION THREAD

**Discussion by sunlightrain** [1Panel-dev/1Panel#5034] [Q&A]
[root@localhost ~]# curl -sSL https://resource.fit2cloud.com/1panel/package/quick_start.sh -o quick_start.sh && sh quick_start.sh
开始下载 1Panel v1.10.7-lts 版本在线安装包
安装包下载地址： https://resource.fit2cloud.com/1panel/package/stable/v1.10.7-lts/release/1panel-v1.10.7-lts-linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 49.6M  100 49.6M    0     0  67.7M      0 --:--:-- --:--:-- --:--:-- 67.6M
1panel-v1.10.7-lts-linux-amd64/1panel.service
1panel-v1.10.7-lts-linux-amd64/1pctl
1panel-v1.10.7-lts-linux-amd64/LICENSE
1panel-v1.10.7-lts-linux-amd64/README.md
1panel-v1.10.7-lts-linux-amd64/install.sh
1panel-v1.10.7-lts-linux-amd64/1panel

 ██╗    ██████╗  █████╗ ███╗   ██╗███████╗██╗     
███║    ██╔══██╗██╔══██╗████╗  ██║██╔════╝██║     
╚██║    ██████╔╝███████║██╔██╗ ██║█████╗  ██║     
 ██║    ██╔═══╝ ██╔══██║██║╚██╗██║██╔══╝  ██║     
 ██║    ██║     ██║  ██║██║ ╚████║███████╗███████╗
 ╚═╝    ╚═╝     ╚═╝  ╚═╝╚═╝  ╚═══╝╚══════╝╚══════╝
[1Panel Log]: ======================= 开始安装 ======================= 
设置 1Panel 安装目录（默认为/opt）：
[1Panel Log]: 您选择的安装路径为 /opt 
[1Panel Log]: ... 在线安装 docker 
选择延迟最低的源 https://mirrors.aliyun.com/docker-ce，延迟为 0.0969017 秒
# Executing docker install script, commit: 6d9743e9656cc56f699a64800b098d5ea5a60020
Packages for RHEL are currently only available for s390x.
[1Panel Log]: ... 启动 docker 
Failed to enable unit: Unit file docker.service does not exist.
Failed to start docker.service: Unit docker.service not found.
[1Panel Log]: docker 安装失败

**Comment by wanghe-fit2cloud** [comment]
参考：https://github.com/1Panel-dev/1Panel/issues/2616
